tests/integration/tests/test_cleanup.py

```diff
@@ -17,21 +19,71 @@
 ]
 
 
+def _assert_paths_not_exist(instance: harness.Instance, paths: List[str]):
```

Comment:
Feels like it would fit better in `test_util.util`

Reply:
Great point!

I've made the function a bit more generic (returns `bool` for each given path) and moved it to `test_util`.